tests/Logging.XUnit.Tests/Integration/PrintableDiagnosticMessage.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Martin Costello, 2018. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+
+using Xunit.Sdk;
+
+namespace MartinCostello.Logging.XUnit.Integration
+{
+    /// <summary>
+    /// See https://github.com/xunit/xunit/pull/2148#issuecomment-839838421
```

Comment:
👍

Reply:
Yeah, that will be one more place for people using Rider to find out about this workaround. 😀